src/pages/ManageResource.tsx

```diff
@@ -8,11 +8,12 @@ const ManageResource = ({
     params: { resourceName }
   }
 }: RouteComponentProps<{ resourceName: string }>) => {
+  console.log({ resourceName });
   return (
     <Wrapper>
       <ResourceHeader name={resourceName} />
-      <div>Provisioning</div>
-      <manifold-resource-details resource-label={resourceName} />
+      <manifold-resource-status resource-name={resourceName} />
+      <manifold-resource-credentials resource-name={resourceName} />
```

Comment:
🤔 I know HTML needs `kebab-case` but in React does it do better with `camelCase`? or does it not matter? Just curious

Reply:
I know this works. not sure if camelCase works.

Reply:
According to [Using Web Components in React](https://reactjs.org/docs/web-components.html) looks like you’re right–they should appear as HTML attributes. I may have messed up a few.